front-end-tools/Jenkinsfile

```diff
@@ -0,0 +1,35 @@
+// Expected parameters:
+// - image_tag: Tag that will be used for the new image.
+// - build_image: Base image that the image being built extends.
+pipeline {
+    agent any
+    environment {
+        image_repo = "concordium/issuer-front-end"
+        image_name = "${image_repo}:${image_tag}"
+    }
+    stages {
+        stage('dockerhub-login') {
+            environment {
+                // Defines 'CRED_USR' and 'CRED_PSW'
+                // (see 'https://www.jenkins.io/doc/book/pipeline/jenkinsfile/#handling-credentials').
+                CRED = credentials('jenkins-dockerhub')
+            }
+            steps {
+                sh 'docker login --username "${CRED_USR}" --password "${CRED_PSW}"'
+            }
+        }
+        stage('build-push') {
+            steps {
+                sh '''\
+                    docker build \
+                      --build-arg build_image="${build_image}" \
+                      --label build_image="${build_image}" \
+                      --tag="${image_name}" \
+                      -f ./front-end-tools/Dockerfile \
+                      .
```

Comment:
Unless we need something outside of the `front-end-tools` directory, it would be better to set the build context to this directory. Since the context is now the root of the repo docker will copy every file in the repo before it starts running the dockerfile, which will become slower as this repo grows (and the more local files you have in the repo)

Reply:
updated